tooling/template-base/package.json

```diff
@@ -11,15 +11,15 @@
     "dev:server": "tsup ./server/index.ts && node ./dist/index.js",
     "build": "npm run build:css && npm run build:vite && npm run build:worker",
     "build:css": "tailwindcss -o ./renderer/tailwind.css",
-    "build:vite": "vite build && vite build --ssr",
+    "build:vite": "vite build",
```

Comment:
got warning when running dev server `[vite-plugin-ssr@0.4.43][Warning] The CLI call `$ vite build --ssr` is superfluous since `$ vite build` also builds the server-side. If you want two separate build steps then use https://vite-plugin-ssr.com/disableAutoFullBuild or use Vite's `build()` API.`

will remove this part of the script.